doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -70,7 +70,7 @@ create_sink ::=
     'FROM' item_name
     'INTO' sink_kafka_connector
     ('FORMAT' sink_format_spec)?
-    ('ENVELOPE' ('DEBEZIUM'|'UPSERT'))?
+    ('ENVELOPE' ('DEBEZIUM'|'UPSERT'))
```

Comment:
Should we enforce this in the parser? If envelope types differ so much between sinks and sources, could make sense to separate the parsing and enums too.

Reply:
Agreed on both counts, but going to merge as is for the wipe week!